apps/mosaico/src/Article/Box.purs

```diff
@@ -32,36 +32,55 @@ autoExpand a = (length $ joinWith mempty a) < 600
 render :: EventHandler -> Props -> JSX
 render setExpanded props =
   DOM.section
-    { className: "boxinfo"
+    { className: "bg-gray-50 border-b-2 border-t-4 border-brand [.article-element\\_\\_reviewbox>&]:border [.article-element\\_\\_reviewbox>&]:border-gray-100"
```

Comment:
This would probably one of the use cases for using `@apply` in CSS, but I wanted to use these as an example on how to use TW's `\_` syntax inside Purescript.

CSS equivalent for the two last classes:
```css
.article-element__reviewbox > .mosaico-boxinfo {
  @apply border border-gray-100;
```